test/open.test.js

```diff
@@ -5,28 +5,6 @@ const { test } = require('node:test')
 const { join } = require('node:path')
 const { createReadStream } = require('..')
 
-test('should open a file', (t, done) => {
-  const pathToFile = join(__dirname, 'files', 'test-file-1.txt')
-  const stream = createReadStream(pathToFile, { autoWatch: false })
-
-  let openEmitted = false
-  stream.on('open', (fd) => {
-    openEmitted = true
-    assert.strictEqual(typeof fd, 'number')
-  })
-
-  let readyEmitted = false
-  stream.on('ready', () => {
-    readyEmitted = true
-  })
-
-  setImmediate(() => {
-    assert.strictEqual(openEmitted, true)
-    assert.strictEqual(readyEmitted, true)
-    done()
-  })
-})
-
```

Comment:
why?

Reply:
The test is failing in the CI. It subscribes on evenets when the file is already opened. Do you know the correct way to test `open` and `ready` events?

Reply:
use https://github.com/mcollina/tspl